prime-router/src/test/kotlin/azure/ActionHistoryTests.kt

```diff
@@ -352,7 +352,8 @@ class ActionHistoryTests {
             Topic.TEST,
             "http://blobUrl",
             TaskAction.send,
-            OffsetDateTime.now()
+            OffsetDateTime.now(),
+            ""
```

Comment:
Any reason we shouldn't put at least a dummy version here? Ditto in other 4 locations.

Reply:
It didn't seem relevant to these specific tests; I only added references where it was needed.